### PR TITLE
Encode variable values for use over URLs

### PR DESCRIPTION
**Pull request by wasade**
This PR allows variable values such as "foo/bar" to be stored. Previously these had been omitted due to overload of the "/" character for use with URLs. These values are now formally encoded using `urllib`.

### CODE REVIEW

File: redbiom/fetch.py [hunk 1]
```diff
@@ -715,3 +692,41 @@ def metadata(where=None, tag=None, restrict_to=None):
...
+def get_sample_values(samples, category, get=None):
+    """Obtain the metadata values associated with the requested samples
+
+    Parameters
+    ----------
+    samples : Iterable of str or None
+        The samples to obtain
+    category : str
+        The category to obtain values for.
+
```
Comment (ElDeveloper) [inline redbiom/fetch.py]: get is missing here

File: redbiom/fetch.py [hunk 2]
```diff
@@ -715,3 +692,41 @@ def metadata(where=None, tag=None, restrict_to=None):
...
+    """
+    import redbiom
+
+    if get is None:
+        config = redbiom.get_config()
+        get = redbiom._requests.make_get(config)
+
+    key = 'category:%s' % category
+    if samples is None:
+        samples = get('metadata', 'HKEYS', key)
```
Comment (ElDeveloper) [inline redbiom/fetch.py]: Note this in the command summary

**Comment by wasade** [comment]
@ElDeveloper polite ping, able to merge by chance?

**Comment by wasade** [comment]
Thanks!

**Pull request merged**